### PR TITLE
Document the feature about generic object count quota 

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1575892

### CODE REVIEW

**Comment by mburke5678** [comment]
@derekwaynecarr Can someone on your team PTAL. This was a 3.9 feature we missed.
Trello:
https://trello.com/c/ayBfZlz8/1033-5-upstream-generic-object-count-quota

**Comment by wanghaoran1988** [comment]
@mburke5678 I think we can add this part to the `Resources Managed by Quota` part like k8s doc structure. **https://kubernetes.io/docs/concepts/policy/resource-quotas/#object-count-quota**

**Comment by mburke5678** [comment]
@wanghaoran1988 PTAL. I moved some information on `count/<resource>.<group>` syntax higher and left the `oc create quota --hard=count/<resource>.<group>=<quota>,count/<resource>.<group>=<quota>` command in the Create a Quota section. 
Is this acceptable?

**Comment by wanghaoran1988** [comment]
@mburke5678 This does make sense to me.
/lgtm

**Comment by mburke5678** [comment]
@derekwaynecarr BUMP PTAL

**Comment by mburke5678** [comment]
@deads2k PTAL

**Comment by mburke5678** [comment]
@deads2k PTAL

**Comment by deads2k** [comment]
> @deads2k PTAL

vacation. lgtm

**Comment by mburke5678** [comment]
@deads2k Apologies! Didn't know you were on holiday. Thank you for taking the time to look. Enjoy!

**Comment by mburke5678** [comment]
@wanghaoran1988 @deads2k 
Are these quotas cluster-wide, where the quota limits a resource in each project in the cluster? The user does not specify a quota for each project?

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: admin_guide/quota.adoc [hunk 1]
```diff
@@ -130,6 +130,16 @@ exceed this value.
 |===
 // end::admin_quota_resources_managed[]
 
+You can object count quota these standard namespaced resource types using the `count/<resource>.<group>` syntax
```
Comment (kalexand-rh) [inline admin_guide/quota.adoc]: maybe s/object count quota/assign an object count quota

File: admin_guide/quota.adoc [hunk 2]
```diff
@@ -130,6 +130,16 @@ exceed this value.
...
 // end::admin_quota_resources_managed[]
 
+You can object count quota these standard namespaced resource types using the `count/<resource>.<group>` syntax 
+while xref:create-a-quota[creating a quota]. 
+
+----
+$ oc create quota <name> --hard=count/<resource>.<group>=<quota> <1>
+----
+
+<1> `<resource>` is the name of the resource and `<group>` is the API group, if applicable.
```
Comment (kalexand-rh) [inline admin_guide/quota.adoc]: s/resource/resource,

File: admin_guide/quota.adoc [hunk 3]
```diff
@@ -401,8 +412,8 @@ spec:
 [[create-a-quota]]
 == Creating a Quota
 
-To create a quota, first define the quota to your specifications in a file, for
-example as seen in
+To create a quota, first define the quota to your specifications in a file, 
+for example as seen in
```
Comment (kalexand-rh) [inline admin_guide/quota.adoc]: maybe s/for example as seen in/such as the

**Review by kalexand-rh — COMMENTED** [review]
The couple of obligatory nitpicks aside, this sounds good to me!

File: admin_guide/quota.adoc (hunk 3 above)
Comment (mburke5678) [reply admin_guide/quota.adoc]: @kalexand-rh That middle phrase is awkward: _to your specification_ sounds odd.
"To create a quota, first define the quota in a file, such as the examples in..."

Comment (kalexand-rh) [reply admin_guide/quota.adoc]: I think that does sound better @mburke5678!

**Comment by wanghaoran1988** [comment]
@mburke5678 I am trying to answer your question, the `ResourceQuota` are created by project admin in each project, or we can use project request template to create the quota, that means when a project admin create a project, a default `ResourceQuota` will be created just like how openshift online and dedicated does.

**Pull request merged**